tests/test_data.py

```diff
@@ -269,7 +270,42 @@ def test_load_electrode_pos(self):
         x, y = load_electrode_pos(data_dir, testfile)
         self.assertEqual(len(x), 244)
         self.assertEqual(len(y), 244)
-    
+
+    def test_map_acq2pos(self):
+        test_signalpathfile = '210910_ecog_signal_path.xlsx'
+        test_mappingfile = '244ch_viventi_ecog_elec_to_pos.xls'
```

Comment:
i would call it a "layout file" so as not to confuse with other maps
```suggestion
        test_layoutfile = '244ch_viventi_ecog_elec_to_pos.xls'
```